Gemfile

```diff
@@ -10,6 +10,9 @@ gem 'coffee-rails', '~> 4.1.0'
 gem 'jquery-rails'
 gem 'twitter-bootstrap-rails'
 
+# allow local server to be started on some Windows machines
```

Comment:
We should move this into the development & test groups.  I'm not 100% on where we'll host the app but it certainly won't be on a Windows environment.  So, no need to include that gem in production.